include/cxxopts.hpp

```diff
@@ -210,7 +210,7 @@ namespace cxxopts
   T
   toLocalString(T&& t)
   {
-    return t;
+    return std::move(t);
```

Comment:
That should work as well. Clang suggested `std::move(t)`.

Reply:
You don know the difference between the two right? Unconditionally moving from a forwarding reference is rarely a good Idea.

Reply:
I haven't checked thoroughly but didn't see any problems from scrolling over it.

Reply:
Yeah that should be `std::forward<T>(t)`. I think if a `const T&` was passed it would be invalid to try to move it.

Reply:
Actually,  calling move on `const T&` is not a problem, as  a `const T&&` will be (silently) copied instead of moved. 

The problem is when a non-const l-value (like a local variable) is passed and the signature becomes `(T& t)`.